src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
I have 2 questions:

1. Why did you rename value.jelly to value.jelly to Base64EncodedStringParameterValue.jelly ?
2. If such `div` element is needed for handling parameters as an array by rebuild plugin, I think that dedicated jelly file should be provided. Because this plugin itself don't need it actually. WDYT?

I have concern that HTML structure for displaying this plugin own object has strong dependency with other plugin.